nautobot/core/views/mixins.py

```diff
@@ -1387,3 +1388,106 @@ def notes(self, request, *args, **kwargs):
             "active_tab": "notes",
         }
         return Response(data)
+
+
+class ComponentCreateViewMixin(NautobotViewSetMixin, mixins.CreateModelMixin):
+    create_form_class: Optional[type[Form]] = None  # TODO: required, declared Optional only to avoid a breaking change
+    form_class: Optional[type[Form]] = None  # TODO: required, declared Optional only to avoid a breaking change
```

Comment:
Since this is new code, you probably could address the copy-paste `TODO` here and just make these required fields, i.e.:

```suggestion
    create_form_class: type[Form]
    form_class: type[Form]
```